File: features/digital/common_digital/src/main/java/com/tokopedia/common_digital/cart/data/entity/requestbody/RequestBodyIdentifier.java
package com.tokopedia.common_digital.cart.data.entity.requestbody;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by Rizky on 27/08/18.
 */
public class RequestBodyIdentifier {

    @SerializedName("user_id")
    @Expose
    private String userId;
    @SerializedName("device_token")
    @Expose
    private String deviceToken;
    @SerializedName("os_type")
    @Expose
    private String osType;

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public void setDeviceToken(String deviceToken) {
        this.deviceToken = deviceToken;
    }

    public void setOsType(String osType) {
        this.osType = osType;
    }

}

File: features/recharge/common_digital/src/main/java/com/tokopedia/common_digital/cart/data/entity/requestbody/RequestBodyIdentifier.kt
package com.tokopedia.common_digital.cart.data.entity.requestbody

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by Rizky on 27/08/18.
 */
class RequestBodyIdentifier {

    @SerializedName("user_id")
    @Expose
    var userId: String? = null
    @SerializedName("device_token")
    @Expose
    var deviceToken: String? = null
    @SerializedName("os_type")
    @Expose
    var osType: String? = null
}
